### PR TITLE
Feature/email

### PR DESCRIPTION
**Pull request by devmarcstorm**

### CODE REVIEW

**Review by C3MO — COMMENTED** [review]
looks good to me.

File: docs/experiments_and_process_variables.md [hunk 1]
```diff
@@ -28,11 +28,13 @@
...
-|---------------------|-----------------------------------|-------------|--------|----------|---------------|
-| pv_string           | String(100)                       | True        | True   | False    | ---           |
-| human_readable_name | String(100)                       | False       | False  | True     | ---           |
-| experiment_short_id | ForeignKey("experiment.short_id") | False       | False  | False    | ---           |
+| Column name           | type                                 | primary key | unique | nullable | default value |
+|-----------------------|--------------------------------------|-------------|--------|----------|---------------|
+| pv_string             | String(100)                          | True        | True   | False    | ---           |
+| human_readable_name   | String(100)                          | False       | False  | True     | ---           |
+| experiment_short_id   | ForeignKey("experiment.short_id")    | False       | False  | False    | ---           |
+| default_threshold_min | db.Column(db.Integer, nullable=True) | False       | False  | True     | ---           |
```
Comment (devmarcstorm) [inline docs/experiments_and_process_variables.md]: Please remove ", nullable=True" for default_threshold_min and default_threshold_max

File: rtp_backend/apps/experiments/views.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
     respond_with_404,
 )
 from rtp_backend.apps.utilities.user_created_data import get_request_dict
+from sqlalchemy import Integer
```
Comment (devmarcstorm) [inline rtp_backend/apps/experiments/views.py]: Not used

File: docs/experiments_and_process_variables.md (hunk 1 above)
Comment (C3MO) [reply docs/experiments_and_process_variables.md]: done

**Review by C3MO — COMMENTED** [review]
looks really good to me.

**Pull request merged**